roles/runtime/tasks/initialize_base.yml

```diff
@@ -152,7 +152,8 @@
         base_instance_group: "{{ run__ml_k8s_request_base }}"
         config:
           name: "{{ __ml_config.name | default([run__namespace_cdp, __ml_config.suffix | default(include.suffix) | default(run__ml_suffix)] | join('-')) }}"
-          nfs: "{{ __ml_config.nfs | default(run__azure_nfs_mount | default(omit)) }}"
+          nfs: "{{ __ml_config.nfs | default(run__azure_netapp_nfs_mount | default(omit)) }}"
+          nfs_version: "{{ __ml_config.nfs_version | default(run__azure_netapp_nfs_version if __ml_config.nfs is defined or run__azure_netapp_nfs_mount is defined | default(omit)) }}"
```

Comment:
When testing `teardown` I get an error on this task that is related to the if expression on this line:

**Error message (snippet):**

```
TASK [cloudera.exe.runtime : Construct CDP ML Workspace configurations] ********
task path: /runner/project/ansible_dev/collections/ansible_collections/cloudera/exe/roles/runtime/tasks/initialize_base.yml:144
Wednesday 30 March 2022  14:02:34 +0000 (0:00:00.170)       0:01:02.875 *******
fatal: [localhost]: FAILED! => {
    "msg": "The task includes an option with an undefined variable. The error was: <snipped>....: 
the inline if-expression on line 1 evaluated to false and no else section was defined.
}
```

**CML definition:**

```
ml:
  definitions:
    - name: "{{ name_prefix }}-default-ml"
      tls: yes
      monitoring: yes
      governance: yes
      metrics: yes
      public_loadbalancer: yes
      storage: yes
      instance_groups:
        - name: cml
          instanceCount: 0
          instanceType: "Standard_DS5_v2"
          rootVolume:
            size: 100
```